packages/tupaia-web/src/components/AuthModal.tsx

```diff
@@ -7,6 +7,7 @@ import React, { ReactNode } from 'react';
 import styled from 'styled-components';
 import { Typography } from '@material-ui/core';
 import { Button } from '@tupaia/ui-components';
+import { ErrorBoundary } from '../../../ui-components/src/components/ErrorBoundary';
```

Comment:
There is a rule in mono-repos where if you are importing modules from another package, you use the workspace name rather than doing a relative import. So to import the ErrorBoundary from ui-components, you would do `import { ErrorBoundary } from '@tupaia/ui-components';`